internal/app_config.go

```diff
@@ -80,6 +80,15 @@ func (c AppConfig) Get(key string) string {
 	return ""
 }
 
+func (c AppConfig) GetOrDie(key string) string {
+	val := c.Get(key)
+	if len(val) == 0 {
+		fmt.Println("ERROR: Missing", key, "in sota.toml")
+		os.Exit(1)
+	}
+	return val
+}
+
```

Comment:
Nitpick: can be part of the first commit, right?

Reply:
> I guess some of our users would want to put their sota.toml of a read-only partition.

The sota.toml itself created by lmp-dr pretty much has to be writable.